### PR TITLE
Update quick-notes extension

### LINKED ISSUE #12557: [Quick Notes] Add Sections as Notes Folder
### Extension

https://www.raycast.com/biknandy/quick-notes

### Description

It would be nice to add the ability to group notes into folders. 

Visually it would look like a section in a list.

The mechanism for creating sections would be similar to how tags are created, but instead of multiple selection, you could only create/select one section.

It will be possible to additionally add DropDown to the search bar to filter by one section

### Who will benefit from this feature?

A user can have notes for different purposes: work, life, hobbies.

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by biknandy**
Closes #12557

## Description

- New quick add / remove tags using `⌘ + T`
- Quick tag sort from dropdown in search
- Updated new tag shortcut to `⌘ + ⇧ + T`
- Updated filter tag to `⌘ + F`
- Better word count in metadata to account for MD
- Update README and switch some imgs

## Screencast

https://github.com/raycast/extensions/assets/68867182/51e87e39-daef-4e55-8829-1e497de979ae

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/quick-notes/CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,21 @@
...
+- New quick add / remove tags using `⌘ + T`
+- Quick tag sort from dropdown in search
+- Updated new tag shortcut to `⌘ + ⇧ + T`
+- Updated filter tag to `^ + T`
+- Better word count in metadata to account for MD
+
+## [Bugfix] - 2024-05-24
+
+- Remove create button on the sort action, unintended
+
```
Comment (biknandy) [inline extensions/quick-notes/CHANGELOG.md:14]: Looks like there's some issue pulling from main here

**Comment by pernielsentikaer** [comment]
Is everything as expected now @biknandy 🙂

**Comment by biknandy** [comment]
@pernielsentikaer yup looks good thanks!

**Comment by biknandy** [comment]
@pernielsentikaer This one good to go? (no rush or anything just wanted to make sure it was still in your backlog!)

**Pull request merged**